helm-app/templates/chat-service-db-service.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.chatServiceDb.service.name }}
+  namespace: {{ .Values.namespace }}
+spec:
+  selector:
+    app: {{ .Values.chatServiceDb.selector.app }}
+  ports:
+  - protocol: TCP
+    port: {{ .Values.chatServiceDb.service.port }}
+    targetPort: {{ .Values.chatServiceDb.service.targetPort }}
+  type: {{ .Values.chatServiceDb.service.type }}

```

Comment:
If i keep the type as `NodePort`, what will happen?

Reply:
Fixed.